colorbleed/plugins/maya/create/colorbleed_vrayscene.py

```diff
@@ -0,0 +1,38 @@
+from collections import OrderedDict
+
+import avalon.maya
+
+
+class CreateVRayScene(avalon.maya.Creator):
+
+    label = "VRay Scene"
+    family = "colorbleed.vrayscene"
+    icon = "cubes"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVRayScene, self).__init__(*args, **kwargs)
+
+        # We won't be publishing this one
+        self.data["id"] = "avalon.vrayscene"
+
+        # We don't need subset or asset attributes
+        self.data.pop("subset", None)
+        self.data.pop("asset", None)
+
+        data = OrderedDict(**self.data)
+
+        data["camera"] = self._get_cameras()
```

Comment:
@aardschok does this work? This means it would just end up become a formatted list in the string like `["cam1", "cam2"]` over [here](https://github.com/Colorbleed/colorbleed-config/pull/150/files#diff-abe4cfcd94241bfaed3e0e5865835bb0R195)

Reply:
No. The `lib.read` function returns the string of the index which has been set.

This example has two renderable cameras added.
```
lib.read("vraysceneDefault")
# Result: {u'suspendPublishJob': False, u'suspendRenderJob': False, u'family': u'colorbleed.vrayscene', u'pools': u'', u'extendFrames': False, u'camera': u'perspShape', u'active': True, u'id': u'avalon.vrayscene'} # 
setAttr "vraysceneDefault.camera" 1;
lib.read("vraysceneDefault")
# Result: {u'suspendPublishJob': False, u'suspendRenderJob': False, u'family': u'colorbleed.vrayscene', u'pools': u'', u'extendFrames': False, u'camera': u'perspShape2', u'active': True, u'id': u'avalon.vrayscene'} # 
```